hack-kni/verify-commits.sh

```diff
@@ -48,7 +48,7 @@ git log --oneline "$latest_upstream_commit"..HEAD
 echo
 
 # list commits
-for commitish in $(git log --oneline --no-merges "$latest_upstream_commit"..HEAD | cut -d' ' -f 1); do
+for commitish in $(git log --oneline --no-merges "$latest_upstream_commit"..HEAD~ | cut -d' ' -f 1); do
```

Comment:
The ```--no-merges``` flag should take care of that, so ```HEAD~``` won't cause skipping the check of the last commit?

Reply:
yes, this is just wrong. The fix (if needed) is elsewhere